pkg/set/set.go

```diff
@@ -0,0 +1,15 @@
+package set
```

Comment:
For directory structure under `cmd/` it makes sense to have `set/` or `get/` subfolders. But under `pkg/` I think we can have a different directory structure that is more convenient for each component. For example if we had `pkg/rook/osd/recovery.go`, then related methods for recovery Get and Set could be included in the same implementation go file, which makes it simpler to share arguments or helpers for the same feature.

Reply:
+1

Reply:
done